install_config/install/docker_registry.adoc

```diff
@@ -171,17 +171,28 @@ endif::[]
 
 [[viewing-logs]]
 
-==== Setting storage quota
-To protect the storage from being accidentally filled up or to prevent users from abusing the space,
-you may set size constraints on images, image streams and projects.
+==== Setting Storage Quota
+To protect the storage from being accidentally filled up or to prevent users
+from abusing the space, you may set size constraints on images, image streams,
+and projects.
 
-link:../../dev_guide/quota.html#create-a-quota[Create a quota object] on particular namespace with
-one or more of the following resources:
+link:../../dev_guide/quota.html#create-a-quota[Create a quota object] on
+particular namespace with one or more of the following resources:
 
-- `openshift.io/imagesize` - to constrain the maximum size of an image
-- `openshift.io/imagestreamsize` - to limit the total images size tagged in one image stream
-                                   (aka repository)
-- `openshift.io/projectimagessize` - to limit the total images size per project
+[cols="1,4",options="header"]
+|===
+|Resource|Purpose
+|`openshift.io/imagesize`
+|To constrain the maximum size of an image
+
+|`openshift.io/imagestreamsize`
+|To limit the total images' size tagged in one image stream (also known as a
+repository)
+
+|`openshift.io/projectimagessize`
+|To limit the total images size per project
```

Comment:
I thought the projectimagessize meant more like "To limit the total combined size of all images per project", but I could be wrong.